08LessonEight/homework/exercise-06.html

```diff
@@ -2,9 +2,40 @@
 <html>
 
 <head>
-    <title>Exercise 06</title>
-</head>
+   <title>Safe & Secure Delivery</title>
+   <link rel="stylesheet" href="css/exercise-06-style.css" </head>
 
-<body></body>
+   <body>
+   <section id="clear-fix">
+       <br>
+       <div class="left">
+           <img src="img/truck.jpg">
+           <h2>Safe & Secure Delivery</h2>

```

Comment:
Remember to use `&amp;` for the `&` symbol - this will prevent errors while your HTML is being parsed